src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -35,7 +41,7 @@ export const AGGREGATIONS = {
     isSortable: true,
   },
   sum: {
-    type: ['transaction.duration'],
+    type: ['duration'],
```

Comment:
@leedongwei @markstory can either of you confirm if this is the right type?

Reply:
Ok thanks. I referred to this specific line of code to see what the mapping looks like https://github.com/getsentry/sentry/blob/ff6382011bab16dee8af36141fbbdf6517cb54b1/src/sentry/static/sentry/app/views/eventsV2/table/tableModalEditColumn.tsx#L197

and made the correction.